### ISSUE TITLE
RCLONE spamming syslog messages

### ISSUE THREAD

**Issue by rubenkr** [rclone/rclone#5931]
Hi everyone,

first let me thank you for your contribution to this awesome project and looking at my issue. I want to be precise as possible.

**TL;DR:**
- Installed rclone from AUR packages (yay -S rclone) on archlinux x86_64
- Tried rclone with GoogleDrive and stopped the mounting, deleted the config
- Syslog show massive load of log file (see sample below) with rclone help messages every second!
- Even with rclone removed and no process running syslog is still full of it
- I deleted the unit file in /etc/systemd/system/rclone.service, reloaded daemon and reset failed

#### What is the problem you are having with rclone?
after install rclone v.1.57.0 I have massive load of rclone messages in my syslog.

#### What is your rclone version (output from `rclone version`)
`2022/01/20 08:52:31 DEBUG : rclone: Version "v1.57.0" starting with parameters ["rclone" "-vv"]`

➜  log which rclone
/usr/bin/rclone

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)
uname -a
`Linux T450 5.16.1-arch1-1 #1 SMP PREEMPT Sun, 16 Jan 2022 11:39:23 +0000 x86_64 GNU/Linux`

➜  log cat /etc/os-release
```
NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://bugs.archlinux.org/"
LOGO=archlinux-logo
```

#### Which cloud storage system are you using? (e.g. Google Drive)
Google Drive from the rclone config (without API code)

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)
I stopped rclone and deleted rclone via yay -R rclone. Still I receive syslog messages

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)

 sudo  syslog-ng -Fdev
`[2022-01-20T08:58:36.726730] Outgoing message; message='Jan 10 07:46:18 T450 rclone[3993120]:       --no-checksum                            Don\'t compare checksums on up/download\x0a'
[2022-01-20T08:58:36.726746] Outgoing message; message='Jan 10 07:46:18 T450 rclone[3993120]:       --no-modtime                             Don\'t read/write the modification time (can speed things up)\x0a'
[2022-01-20T08:58:36.726761] Outgoing message; message='Jan 10 07:46:18 T450 rclone[3993120]:       --no-seek                                Don\'t allow seeking in files\x0a'
[2022-01-20T08:58:36.726802] Incoming log entry from journal; message='      --noappledouble                          Ignore Apple Double (._) and .DS_Store files (supported on OSX only) (default true)'
[2022-01-20T08:58:36.726819] json-parser(): no marker at the beginning of the message, skipping JSON parsing ; input='      --noappledouble                          Ignore Apple Double (._) and .DS_Store files (supported on OSX only) (default true)', marker='@cee:'
[2022-01-20T08:58:36.726829] json-parser(): no marker at the beginning of the message, skipping JSON parsing ; input='      --noappledouble                          Ignore Apple Double (._) and .DS_Store files (supported on OSX only) (default true)', marker='@cim:'
[2022-01-20T08:58:36.726870] Incoming log entry from journal; message='      --noapplexattr                           Ignore all "com.apple.*" extended attributes (supported on OSX only)'
[2022-01-20T08:58:36.726885] json-parser(): no marker at the beginning of the message, skipping JSON parsing ; input='      --noapplexattr                           Ignore all "com.apple.*" extended attributes (supported on OSX only)', marker='@cee:'
[2022-01-20T08:58:36.726893] json-parser(): no marker at the beginning of the message, skipping JSON parsing ; input='      --noapplexattr                           Ignore all "com.apple.*" extended attributes (supported on OSX only)', marker='@cim:'
[2022-01-20T08:58:36.753231] Outgoing message; message='Jan 10 07:46:18 T450 rclone[3993120]:       --noappledouble                          Ignore Apple Double (._) and .DS_Store files (supported on OSX only) (default true)\x0a'
[2022-01-20T08:58:36.753272] Outgoing message; message='Jan 10 07:46:18 T450 rclone[3993120]:       --noapplexattr                           Ignore all "com.apple.*" extended attributes (supported on OSX only)\x0a'
[2022-01-20T08:58:36.753303] Outgoing message; message='Jan 10 07:46:18 T450 rclone[3993120]:       --noappledouble                          Ignore Apple Double (._) and .DS_Store files (supported on OSX only) (default true)\x0a'
[2022-01-20T08:58:36.753320] Outgoing message; message='Jan 10 07:46:18 T450 rclone[3993120]:       --noapplexattr                           Ignore all "com.apple.*" extended attributes (supported on OSX only)\x0a'
[2022-01-20T08:58:36.753361] Outgoing message; message='Jan 10 07:46:18 T450 rclone[3993120]:       --noappledouble                          Ignore Apple Double (._) and .DS_Store files (supported on OSX only) (default true)\x0a'
[2022-01-20T08:58:36.753382] Outgoing message; message='Jan 10 07:46:18 T450 rclone[3993120]:       --noapplexattr                           Ignore all "com.apple.*" extended attributes (supported on OSX only)\x0a'
[2022-01-20T08:58:36.753905] Running application hooks; hook='4'
`

THANK YOU very much for every hint how to fix this in advance.

**Comment by albertony** [comment]
Is the case that you are running with `-vv`? Then you have enabled full [verbosity](https://rclone.org/docs/#v-vv-verbose), which includes a lot of debug output. Reducing to `-v`, or removing it entirely, will then improve the situation.

**Comment by animosity22** [comment]
As @albertony mentioned, you installed it and ran it with -vv.

We try to keep issues for bug reports and this is a general use question.

Please use the forums for general use.

https://forum.rclone.org/

For anything else.
